.ci/unit-tests/inspec.sh

```diff
@@ -3,4 +3,9 @@
 set -e
 set -x
 
-echo 'TODO slevenick write tests'
\ No newline at end of file
+pushd "magic-modules/build/inspec/libraries"
+
+bundle install
+rspec -I . ../test/unit/*
```

Comment:
Just our of curiosity, what's the -I for? Is that because there's not a spec/ folder?

Reply:
My understanding is that -I adds the following directory to the load path. So we are navigating to the directory containing the InSpec resources and using -I to add `.` to the load path so that the resources are accessible from the tests. I'm not super good on ruby dependency management, so if there is a better way I'm happy to hear suggestions

Reply:
And then it executes everything in `../test/unit`